aws_serverless/template.yaml

```diff
@@ -81,6 +81,37 @@ Resources:
             Path: /api/session/create
             Method: post
 
+  GetSessionRestaurantsFunction:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      CodeUri: restaurant/get/
+      Handler: app.lambda_handler
+      Runtime: python3.9
+      Environment:
+        Variables:
+          YELP_KEY: '{{resolve:secretsmanager:arn:aws:secretsmanager:us-east-1:600091550595:secret:yelp_key-M2asns:SecretString:key}}'
+      Policies:
+        - AWSSecretsManagerGetSecretValuePolicy:
+            SecretArn:
+              arn:aws:secretsmanager:us-east-1:600091550595:secret:yelp_key-M2asns
+        - DynamoDBWritePolicy:
+            TableName:
+              Grubfinder_Session
+        - DynamoDBWritePolicy:
+            TableName:
+              Grubfinder_User
+        - DynamoDBWritePolicy:
```

Comment:
Let's change this to "DynamoDBReadPolicy" for the Grubfinder_Restaurant policy and get rid of the others. Best to limit the scope of a lambda to limit attack surfaces.

Reply:
Oh shoot! Good catch!